src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -1,15 +1,15 @@
 @doc raw"""
-    CompressibleNavierStokesDiffusion2D(gamma, Re, Pr, Ma_inf, equations,
+    CompressibleNavierStokesDiffusion2D(gamma, mu, Pr, R, equations,
                                         gradient_variables=GradientVariablesPrimitive())
 
 These equations contain the diffusion (i.e. parabolic) terms applied
 to mass, momenta, and total energy together with the advective terms from
 the [`CompressibleEulerEquations2D`](@ref).
 
 - `gamma`: adiabatic constant,
-- `Re`: Reynolds number,
+- `mu`: viscosity,
 - `Pr`: Prandtl number,
-- `Ma_inf`: free-stream Mach number
+- `R`: gas constant
```

Comment:
Since we give a default value for R below, should we also mention it here?

Reply:
Yeah, we can mention it here. Currently, the `R` actually cancels away with how temperature and `kappa` in Fick's law are defined. But it could be useful to have around in the future if we (or somebody) wants another type of viscosity that depends on temperature (like Sutherland or power law).

Reply:
I think it would be good to put this at least in a comment (or in a docstring), including that the value currently has no effect.

OTOH, if it is not used at all right now, maybe we should just omit it? I don't like putting something in that has no effect and thus cannot be tested, but instead might confuse someone because changing it to a different value will have no visible effects on the solution. Thus maybe just remove it for now and leave a comment about why and how to put it back in if needed?

Reply:
Even though it is not "doing" anything, I vote to keep it because it makes the implementation directly match formulas from the literature. For example, the temperature is `T = p / ( rho * R )` for the ideal gas. If we do not have `R` (analogous to setting it equal to 1) then it might cause confusion. But I do not have a super strong opinion on this.

Reply:
That's also a reasonable argument - but I would prefer mentioning this in the docstring at least. I don't have a strong opinion on this, though

Reply:
Okay, I will update the docstrings / comments and take it out.

Reply:
OK, thanks. I do not have strong feelings either, thus as long as there is some documentation available, I think it's fine.